src/test/java/org/commcare/formplayer/tests/RepeatTests.java

```diff
@@ -44,7 +44,7 @@ public void testRepeatNonCountedSimple() throws Exception {
         assert (tree.length == 2);
         QuestionBean dummyNode = tree[1];
         assertEquals("false", dummyNode.getExists());
-        assertEquals("Add a new question3", dummyNode.getAddChoice());
+        assertEquals("Add Empty Caption Label", dummyNode.getAddChoice());
```

Comment:
I think it's worth adding back a test that checks the default text is correct

Reply:
Added in https://github.com/dimagi/formplayer/pull/1615/commits/3c381eb5e6dc741bf35097b1a68c9b0137cbee05